explainaboard/analysis/analyses_test.py

```diff
@@ -161,18 +162,18 @@ def test_generate_report(self) -> None:
             level="bar",
             features=("feat1", "feat2"),
             combo_counts=[
-                (("aaa", "bbb"), 123),
-                (("iii", "jjj"), 456),
-                (("xxx", "yyy"), 789),
+                ComboOccurence(("aaa", "bbb"), list(range(3))),
+                ComboOccurence(("iii", "jjj"), list(range(3, 6))),
+                ComboOccurence(("xxx", "yyy"), list(range(6, 9))),
             ],
         )
         report = textwrap.dedent(
             """\
             feature combos for feat1, feat2
             feat1\tfeat2\t#
-            aaa\tbbb\t123
-            iii\tjjj\t456
-            xxx\tyyy\t789
+            aaa\tbbb\t[0, 1, 2]
```

Comment:
This should also be set to `ComboOccurrence.total_count`, not `ComboOccurence.sample_ids`.